src/shared/src/BankSystem.Shared.WebApiDefaults/docs/authentication.md

```
@@ -0,0 +1,257 @@
+// Enable detailed logging
+"Logging": {
+  "LogLevel": {
+    "BankSystem.Shared.WebApiDefaults.Authentication": "Debug",
+    "BankSystem.Shared.WebApiDefaults.Interceptors": "Debug"
+  }
+}
+```
+
+### Log Examples
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)